component/libfuse/libfuse2_handler.go

```diff
@@ -428,6 +428,13 @@ func libfuse2_readdir(_ *C.char, buf unsafe.Pointer, filler C.fuse_fill_dir_t, o
 			}
 		}
 
+		if off_64 == 0 {
+			flags := internal.NewDirBitMap()
+			flags.Set(internal.PropFlagModeDefault)
+			attrs = append(attrs, &internal.ObjAttr{Flags: flags, Name: "."})
```

Comment:
We are still adding it in between here. Attr slice has already elements in place, and we are appending to that so . and .. will come towards the end of this slice, while there might be follow up calls to read this same directory